src/main/java/gov/cabinetoffice/gapfindapiadmin/services/ApiKeyService.java

```diff
@@ -73,8 +75,13 @@ public List<GapApiKey> getApiKeysForSelectedFundingOrganisations(List<String> se
         return Optional.ofNullable(selectedFundingOrgName)
```

Comment:
Think this query could be a bit more efficient by only returning those that match the funding ID, rather than looping through the entire database

Reply:
I know you haven't changed that but I've only jus spotted it there and it's good to stay on top of things

Reply:
it makes sense